### PR TITLE
Dd 62 menu mobile design

### PR DESCRIPTION
**Pull request by rafal-sereda-droptica**

### CODE REVIEW

File: themes/custom/droopler_theme/templates/field/field.html.twig [hunk 1]
```diff
@@ -52,29 +52,32 @@
...
-    {% endif %}
-    {% for item in items %}
-      <div{{ item.attributes.addClass('field__item') }}>{{ item.content }}</div>
-    {% endfor %}
-    {% if multiple %}
-      </div>
-    {% endif %}
-  </div>
-{% endif %}
+{% endblock %}
```
Comment (rafal-sereda-droptica) [inline themes/custom/droopler_theme/templates/field/field.html.twig]: here i only added {% block %} and {% endblock %}
the text intend did this mess.

File: themes/custom/droopler_theme/js/main-menu.js [hunk 2]
```diff
@@ -11,29 +11,47 @@
...
+            $(this).toggleClass('open').parent().find(blockContentClass).slideToggle();
+
+            return false;
           }
         });
       }
     }
   };
 
+  Drupal.behaviors.mainMenuMobileNavbarListener = {
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/js/main-menu.js]: Please add some comments what is going on, at least the behaviors descriptions

File: themes/custom/droopler_theme/templates/field/field--block-content--field-d-long-text.html.twig [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% extends "field--text.html.twig" %}
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/templates/field/field--block-content--field-d-long-text.html.twig]: Short file comment explaining what it is

File: themes/custom/droopler_theme/templates/navigation/we-megamenu-li.html.twig [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<li {{ attributes }}>
```
Comment (grzegorz-pietrzak-droptica) [inline themes/custom/droopler_theme/templates/navigation/we-megamenu-li.html.twig]: File comment

**Pull request merged**